README.md

```diff
@@ -31,4 +31,36 @@ helm uninstall my-release
 
 ## Available charts
 
-* [1Password SCIM bridge](https://github.com/1Password/op-scim-helm/tree/main/charts/op-scim-bridge)
\ No newline at end of file
+* [1Password SCIM bridge](https://github.com/1Password/op-scim-helm/tree/main/charts/op-scim-bridge)
+
+## Resource Recommendations
```

Comment:
What do you think about:
1. providing instructions and/or an example on how you can override these values in Helm commands, and
2. link to the Kubernetes docs that describe how these resource units are measured?

Reply:
Agree!